packages/toggleInputList/components/ToggleInputList.tsx

```diff
@@ -68,54 +68,28 @@ export interface ToggleInputListProps {
   /**
    * Sets the current appearance of the label component. This defaults to InputAppearance.Standard, but supports `InputAppearance.Error` & `InputAppearance.Success` appearances as well.
    */
-  labelAppearance: InputAppearance;
+  labelAppearance?: InputAppearance;
```

Comment:
Why do we change this property from being required to being optional?

Reply:
Once you turn the class component into a function component it will ask for the required prop even if you have defined a default value.

As `labelAppearence` has a default value `InputAppearance.Standard` I changed the prop to optional.

Reply:
Thank you for the explanation, this is fine then. 👍